README.md

```diff
@@ -99,7 +99,7 @@ class TracingConfiguration {
                 tracer,
                 Integer.MIN_VALUE,               // Order
                 Pattern.compile(""),             // Skip pattern
-                Collections.singletonList("/*"), // URL patterns
+                Collections.emptyList(),         // URL patterns, empty list means all
```

Comment:
If empty means all - how one can disable webflux tracing for all URLs?

Reply:
They can disable it using `opentracing.spring.web.enabled = false`.

It's worth noting that the existing behavior for the MVC `ServerTracingAutoConfiguration` is the same: it binds to all URLs if an empty `urlPattern` list is specified.

This is because that is the default behavior in [FilterRegistrationBean](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/web/servlet/FilterRegistrationBean.html):
> When no URL pattern or servlets are specified the filter will be associated to '/*'.